### ISSUE TITLE
Post Compilation - 'RNA_Types' object has no attribute 'TOPBAR_MT_file_new'

### ISSUE THREAD

**Issue by Draise14** [Bforartists/Bforartists#985]
![bforartists_2019-06-28_16-58-46](https://user-images.githubusercontent.com/35357409/60373499-02240480-99c6-11e9-84b8-4c81340d8205.png)

# Issue
I can't compile on my Draises_Branch, nor the Master. 

![bforartists_2019-06-28_18-28-13](https://user-images.githubusercontent.com/35357409/60376163-9e540880-99d2-11e9-86c7-e236a5708174.png)

I have issues with a bunch of addons, SDS funcitonality, and all of them usually have an issue with the follwing (it's a huge bunch of errors): **TOPBAR_MT_file_import**, **IMAGE_MT_uvs**, **RENDER_PT_context**, **TOPBAR_MT_file_export**, **VIEW3D_MT_transform**, **VIEW3D_MT_edit_mesh**, **VIEW3D_MT_view**

Mainly because there is no toolbar: 
**Error, region type 12 missing in - name:"Toolbar", id:23**

Help? 

1. I've deleted and rebuilt the Cmake from scratch
2. I've compiled on Debug and Release
3. I've removed the %appdata% config files
4. I've fetched and pulled and switched to master

Nothing works. What is damaged?

**Comment by Draise14** [comment]
![bforartists_2019-06-28_19-01-09](https://user-images.githubusercontent.com/35357409/60378067-90a67f00-99e2-11e9-9bef-201ba3c208b9.png)

I get other errors trying to compile on the Lite version on a clean Cmake from master. It's all **MT** errors.

**Comment by ReinerBforartists** [comment]
These are Python errors. Python errors does not belong to compile problems. The compilation worked fine. Else you could not start the software. Even when the UI is not showing.

What you get here is a problem in the Python files somewhere. Let me recompile from scratch to check if this happens to me too. Possible that my cache has hidden this problems from me.

**Comment by ReinerBforartists** [comment]
Okay. That's not nice. Does indeed not show the UI.

Investigating ...

**Comment by ReinerBforartists** [comment]
Ah, i see what happened here. I forgot to copy the space_sequencer.py to the repository and to commit the fixes ...

Draise, this is your chance to learn to read the error messages :)

What you do is to have a look at the error message from the top to the bottom. The cause is usually somewhere at the top. Have a look for the mentioned python files.

The first three files cannot be the problem. The first is a warning from the debug build that there is a problem in the C code somewhere. The second two are the init.py, where it moans with nearly every python error with the UI files. The third one then jumped immediately into my face. space_sequencer.py. And you can read in which line the problem is. In line 1005. And it even tells you the problem.     box = col.box() has an indentation problem. Which was the cause. 

![errormessagereading](https://user-images.githubusercontent.com/13919702/60380760-06463580-9a4b-11e9-8c0e-4f749c33ee54.jpg)

There was also another trouble line, which i also fixed. But that's in general how the Python error messages works. It tells you the file, the line number, and most of the times even the problem.

**Comment by ReinerBforartists** [comment]
Fixed in master.

I hope :D

**Comment by Draise14** [comment]
OH man, thanks for the fix. 

I was trying to read it all to find the answer, but it was pages of errors and for some reason I was going from BOTTOM to TOP! So I felt overwhelmed and concluded I must be reading it wrong. I was working under a deadline and I needed the software asap to deliver, and I got clouded by all the cortisol - I jumped back to Alpha 0.5.0 to continue work. 

Next time I'll know better. Thanks for the tips. These types are things are rarely documented.
